source/hardware/controllers/native_instruments_traktor_kontrol_s4_mk3.rst

```diff
@@ -382,7 +397,7 @@ Color of the tempo led when on high takeover
 Keep transport and play button dimmed when off                                                 `InactiveLightsAlwaysBacklit`               true                                                                                                              true/false                                                                            Having this setting on will keep LED always dimmed, even when off, although they may not have a matching color with the deck's color
 Keep the unselected deck button off rather than show its deck color                            `DeckSelectAlwaysBacklit`                   true                                                                                                              true/false                                                                            Having this setting on will keep the LED of the unselected deck dimmed instead of off.
 Keylock on :hwlabel:`SHIFT` + :hwlabel:`MASTER` instead of :hwlabel:`SHIFT` + :hwlabel:`SYNC`  `UseKeylockOnMaster`                        false                                                                                                             true/false                                                                            Use :hwlabel:`SHIFT` + :hwlabel:`MASTER` to toggle the keylock instead of :hwlabel:`SHIFT` + :hwlabel:`SYNC`
-Make the grid button blink when over a detected beat                                           `GridButtonBlinkOverBeat`                   false                                                                                                             true/false                                                                            Make the :hwlabel:`GRID` button blinking when the playback goes over a detected beat
+Make the :hwlabel:`grid` button blink when over a detected beat                                `GridButtonBlinkOverBeat`                   false                                                                                                             true/false                                                                            Make the :hwlabel:`GRID` button blinking when the playback goes over a detected beat
```

Comment:
It is lit when at a beat. Want me to update that?

Reply:
I don't insist, users will notice when scratching over a beat.
Though I would use "flashes" instead of "blinks" (the latter implies repetition IMHO)

I don't mind much, just noticed it.
Change to whatever you think fits.

Reply:
... and let me know when you consider this finished.
It's just polishing after all.

Reply:
Actually, I would appreciate if we could merge that PR here. It's focus was to add the new mixer feature and I think this is done now. Happy to create an issue if you think that document could use some more polishing, perhaps @ywwg could help me with this in a follow up if that's okay?

Reply:
can do

Reply:
Created #650